docs/src/tutorial/cython_tutorial.rst

```diff
@@ -255,49 +326,57 @@ sense for those lines to be yellow. Same for the list comprehension because
 it involves the creation of a Python object. But the line ``if n % i == 0:``, why?
 We can examine the generated C code to understand:
 
-.. figure:: python_division.png
+.. tabs::
+    .. group-tab:: Cython
+
+        .. figure:: python_division.png
+
+    .. group-tab:: Pure Python
+
+        .. figure:: python_division_pure.png
 
 We can see that some checks happen. Because Cython defaults to the
 Python behavior, the language will perform division checks at runtime,
 just like Python does. You can deactivate those checks by using the
 :ref:`compiler directives<compiler-directives>`.
 
-Now let's see if, even if we have division checks, we obtained a boost in speed.
-Let's write the same program, but Python-style:
+Now let's see if we get a speed increase even if there is a division check.
+Let's write the same program, but in Python:
 
 .. literalinclude:: ../../examples/tutorial/cython_tutorial/primes_python.py
+    :caption: primes_python.py / primes_python_compiled.py
 
-It is also possible to take a plain ``.py`` file and to compile it with Cython.
-Let's take ``primes_python``, change the function name to ``primes_python_compiled`` and
-compile it with Cython (without changing the code). We will also change the name of the
-file to ``example_py_cy.py`` to differentiate it from the others.
-Now the ``setup.py`` looks like this::
+It is possible to take a plain ``.py`` file and to compile it with Cython.
```

Comment:
"plain .py file" -> "unannonated .py file" or "untyped .py file" (since the "pure" case kind of is a plain py file anyway)

Reply:
[here](https://github.com/cython/cython/pull/4226/commits/943bc7031b97266dc0eb3817a180628dd8cec5ca)